Dockerfile

```diff
@@ -73,4 +74,8 @@ WORKDIR /
 RUN rm -rf ./azure-cli && \
     dos2unix /root/.bashrc /usr/local/bin/az
 
+# Install kubectl with bash-completion
+RUN az aks install-cli && \
```

Comment:
As mentioned in the comments, I'd rather see this layer added to our Ubuntu image.